.github/labeler.yml

```diff
@@ -1,5 +1,5 @@
 build/ops:
-  - **/CMakeLists.txt
+  - "**/CMakeLists.txt"
```

Comment:
might want to use single quote. like
```yaml
build/ops:
  - '**/CMakeLists.txt'
```

Reply:
AFAIK double quotes work here too. The failure below is because the `pull_request_target` runs on the base source code, which is not fixed yet, and not in the PR code.

Reply:
yeah, i was using a buggy validator.

https://yaml.org/spec/1.2/spec.html#id2787109